ext/opentelemetry-ext-otlpexporter/README.rst

```diff
@@ -0,0 +1,26 @@
+OTLP Exporter
+=============
+
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opentelemetry-ext-otlpexporter.svg
+   :target: https://pypi.org/project/opentelemetry-ext-otlpexporter/
+
+.. FIXME should this exporter export to OTCollector only or to anything that understands OTLP?
+
+This library allows to export data  using OTLP.
+
+Installation
+------------
+
+::
+
+     pip install opentelemetry-ext-otlpexporter
+
+
+References
+----------
+
+* `OpenTelemetry Collector Exporter <https://opentelemetry-python.readthedocs.io/en/latest/ext/otlpexporter/otlpexporter.html>`_
+* `OpenTelemetry Collector <https://github.com/open-telemetry/opentelemetry-collector/>`_
+* `OpenTelemetry <https://opentelemetry.io/>`_
```

Comment:
Would be good to add a link to the OTLP definition

Reply:
Fixed